### PR TITLE
feat(@wallet): Add opensea api key

### PR DESCRIPTION
**Pull request by alaibe**
status-go pr: https://github.com/status-im/status-go/pull/2528

### CODE REVIEW

**Review by richard-ramos — APPROVED** [review]

**Review by jakubgs — CHANGES_REQUESTED** [review]
You're missing changes from `status-jenkins-lib` though.

**Comment by alaibe** [comment]
@jakubgs is there anything I need to do? I think I have to upgrade the version once the PR has landed right?

**Comment by jakubgs** [comment]
You can test the branch from https://github.com/status-im/status-jenkins-lib/pull/38 by specifying it in the `Jenkinsfile`:
https://github.com/status-im/status-desktop/blob/306471be60d3db74a67106028b97f7c40008e5c6/ci/Jenkinsfile.linux#L1
Then once that PR is merged and tagged you can change it back to a proper tag.

**Comment by alaibe** [comment]
Specified in the jenkinsfile

File: ci/Jenkinsfile.combined [hunk 1]
```diff
@@ -1,4 +1,4 @@
-library 'status-jenkins-lib@v1.3.3'
+library 'status-jenkins-lib@6328a0484c8cd331ed8ef58e8ae1400cfaff08aa'
```
Comment (jakubgs) [inline ci/Jenkinsfile.combined:1]: You can just use the branch name, like:
```
library 'status-jenkins-lib@creds/opensea-api-key'
```
But this works too. Once we confirm this works correctly I'll merge that PR and we can use a tag here.

**Comment by jakubgs** [comment]
We can see in the build log that the env variable is being created:
```
14:51:08  Masking supported pattern matches of $INFURA_TOKEN or $OPENSEA_API_KEY or $LINUX_GPG_PRIVATE_KEY_PASS or $LINUX_GPG_PRIVATE_KEY_FILE
```
https://ci.status.im/job/status-desktop/job/branches/job/linux/job/PR-4775/3/console

So that's good.

**Comment by alaibe** [comment]
At first I used a branch then switch to commit as I though I did a mistake, so the correct format is @branch_name ?

**Comment by jakubgs** [comment]
Yeah, the format is essentially `{library_name}@{git_ref}`, and it appears to work:
```
15:42:14  Masking supported pattern matches of $INFURA_TOKEN or $OPENSEA_API_KEY or $MACOS_CODESIGN_IDENT or $MACOS_KEYCHAIN_PASS or $MACOS_KEYCHAIN_FILE
```
https://ci.status.im/job/status-desktop/job/branches/job/macos/job/PR-4775/5/consoleFull

**Comment by jakubgs** [comment]
I will merge the `status-jenkins-lib` PR and tag a release.

**Comment by anastasiyaig** [comment]
they are back!

Screenshot 2022-02-14 at 18 08 36

**Review by anastasiyaig — APPROVED** [review]

File: ci/Jenkinsfile.combined [hunk 2]
```diff
@@ -1,4 +1,4 @@
-library 'status-jenkins-lib@v1.3.3'
+library 'status-jenkins-lib@creds/opensea-api-key'
```
Comment (jakubgs) [inline ci/Jenkinsfile.combined:1]: Please update the references to use the tag.

Comment (alaibe) [reply ci/Jenkinsfile.combined]: yep, updated

**Pull request merged**